Feature/core package file upload

mugnivenko opened the pull request:
https://app.clickup.com/t/1xgnk8a

akeamwoodsND reviewed (changes requested):
Not same label, please use <FormGroup> component inside file upload component
<img width="420" alt="Screenshot 2021-12-14 at 13 01 42" src="https://user-images.githubusercontent.com/89010589/146003294-b85f1ff5-568c-4943-820b-231af77fdc71.png">

akeamwoodsND commented on pages/care-package/[guid]/core-package.js:
```diff
@@ -105,7 +123,19 @@ const CorePackage = () => {
+        fileName: 'socialWorkerCarePlanFileName'
+      });
+
+      if (!data.isS117Client) formData.append('isS117Client', false);
```
why haven't you used getFormData ? can make a modified getFormData that returns false instead of null

akeamwoodsND reviewed (changes requested):
spacing isn't same...
<img width="803" alt="Screenshot 2021-12-14 at 13 09 56" src="https://user-images.githubusercontent.com/89010589/146004464-05d3f6a8-4bdc-4dfa-b5a8-6a6c9abe2897.png">

mugnivenko commented:
1) fixed label ![image](https://user-images.githubusercontent.com/77606518/146009339-f190116a-7999-4ed3-a895-57d1d15f515f.png)

2) Spacing changed

3) added (checkBoolean) field for getFormData
